[2025-04-09T07:11:19.944+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.transform_lastfm manual__2025-04-09T07:11:12.014769+00:00 [queued]>
[2025-04-09T07:11:19.956+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.transform_lastfm manual__2025-04-09T07:11:12.014769+00:00 [queued]>
[2025-04-09T07:11:19.956+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T07:11:20.022+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_lastfm> on 2025-04-09 07:11:12.014769+00:00
[2025-04-09T07:11:20.028+0000] {standard_task_runner.py:60} INFO - Started process 133 to run task
[2025-04-09T07:11:20.032+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'transform_lastfm', 'manual__2025-04-09T07:11:12.014769+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmpnxk3js3e']
[2025-04-09T07:11:20.034+0000] {standard_task_runner.py:88} INFO - Job 81: Subtask transform_lastfm
[2025-04-09T07:11:20.132+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.transform_lastfm manual__2025-04-09T07:11:12.014769+00:00 [running]> on host 6d74a53745cf
[2025-04-09T07:11:20.327+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='transform_lastfm' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T07:11:12.014769+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T07:11:12.014769+00:00'
[2025-04-09T07:11:20.329+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T07:11:20.331+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/transform_lastfm.py']
[2025-04-09T07:11:20.349+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T07:11:24.492+0000] {subprocess.py:93} INFO - ‚ùå Error al transformar Last.fm: [Errno 2] No such file or directory: '../data/processed/lastfm.parquet'
[2025-04-09T07:11:24.625+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T07:11:24.700+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=transform_lastfm, execution_date=20250409T071112, start_date=20250409T071119, end_date=20250409T071124
[2025-04-09T07:11:24.786+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T07:11:24.851+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
